src/main/java/org/highmed/numportal/service/ehrbase/ResponseFilter.java

```diff
@@ -39,7 +42,7 @@ public void initialize() {
 
   public List<QueryResponseData> filterResponse(List<QueryResponseData> queryResponseDataList) {
     if(pathFilters == null){
-      return new ArrayList<>();
+      return queryResponseDataList;
```

Comment:
Can you explain why this functionality has to change?

Reply:
In case there are no filters, the unfiltered data should be returned instead of an empty list. The log message is easy to miss and it is confusing when no data is returned.